zhaquirks/sonoff/zbminir2.py

```diff
@@ -0,0 +1,86 @@
+"""Sonoff ZBMINIR2 - Zigbee Switch."""
+
+from zigpy import types
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+import zigpy.types as t
+from zigpy.zcl import foundation
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class CustomSonoffCluster(CustomCluster):
```

Comment:
I think we can change this to be just `SonoffCluster`. It's a bit shorter.
```suggestion
class SonoffCluster(CustomCluster):
```